src/app/route/route-list/route-list.component.ts

```diff
@@ -34,9 +35,13 @@ export class RouteListComponent {
   user = null;
   selectedRoute: Route;
 
+  @Input() addingPointToRoute: boolean = false;
+  @Input() addingPointToRouteId: string;
```

Comment:
I would change the name. It's vague. I don't know if it refers to ML id or route id. 
maby "pointIdToBeAdded(toRoute)" ?